### ISSUE TITLE
Task - Success notification when logging in/out and saving data

### ISSUE THREAD

**Issue by gggrant** [GCTC-NTGC/TalentCloud#602]
# Description
The 'Save' buttons (Save Diploma/Degree, Save Reference, etc) cause the page to reload with form data. Users with assistive technologies don't receive a cue when this happens, the confirmation is not obvious.

Suggest adding some confirmation text (Your diploma was successfully saved...) which would receive focus when the button is clicked and the page reloads.

**Comment by tristan-orourke** [comment]
Notes from slack conversation:

> You can usually just flash to the session and have a template partial that checks for available flashed messages.
> https://laravel.com/docs/6.x/session#flash-data
> return redirect()->route('settings.edit')->withSuccess('Change of password was successful.');
